From 8cc15658a66230ebfc2346750f443815ae210804 Mon Sep 17 00:00:00 2001
From: "Alexander V. Wolf" <alex.v.wolf@gmail.com>
Date: Tue, 1 Nov 2022 01:35:02 +0700
Subject: [PATCH] Hotfix for bug #2799

---
 src/core/Dithering.hpp | 2 ++
 1 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/src/core/Dithering.hpp b/src/core/Dithering.hpp
index efd3bb08c09..de4d60ff992 100644
--- a/src/core/Dithering.hpp
+++ b/src/core/Dithering.hpp
@@ -42,4 +42,6 @@ Vec3f calcRGBMaxValue(DitheringMode mode);
 
 QString makeDitheringShader();
 
+Q_DECLARE_METATYPE(DitheringMode)
+
 #endif
