diff -Nur orig/lac.desktop mod/lac.desktop
--- orig/lac.desktop	2022-12-09 10:13:18.000000000 +0100
+++ mod/lac.desktop	2022-12-09 10:16:49.748983339 +0100
@@ -1,14 +1,8 @@
 [Desktop Entry]
-Comment=
-Exec=bash lac
+Exec=lac
 GenericName=lac
-Icon=/home/lac/LAC/lac.png
 Name=lac
 NoDisplay=false
-Path[$e]=
 StartupNotify=true
-Terminal=0
-TerminalOptions=
 Type=Application
-X-KDE-SubstituteUID=false
-X-KDE-Username=
+Categories=Game;Simulation;
